spec/game_spec.rb

```diff
@@ -13,12 +13,29 @@
           |__|__|__|
         C |  |  |  |
           |__|__|__|
+
+        Where would you like to play?
+        Enter a row letter and column number. For Example: "A1".
       SQUIGGLY_HEREDOC
 
-      output = StringIO.new
-      Game.run(output)
+      mock_stdout = StringIO.new
+      mock_stdin = StringIO.new
+      allow(mock_stdin).to receive(:gets).and_return("1A", "A1")
+
+      Game.run(mock_stdout, mock_stdin)
+
+      expect(mock_stdout.string).to include(start_message)
+      expect(mock_stdout.string).to include("Hm, that doesn\'t seem quite right")
+    end
+
+    it "exits when user inputs a valid coordinate" do
+      mock_stdout = StringIO.new
+      mock_stdin = StringIO.new
+      allow(mock_stdin).to receive(:gets).and_return("A1")
 
-      expect(output.string).to eq(empty_board)
+      expect {
+        Game.run(mock_stdout, mock_stdin)
+      }.to exit
```

Comment:
Is `exit` an RSpec thing? My expectation, without knowing, is that this is Ruby's `exit` that will halt the spec suite.

Reply:
Thanks for pointing this out! Updated to catch the error. c522627b9df6321dd5fe822111df2b01218d4ee5